tools/ui-components/src/form-group/form-group.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { FormGroup, FormGroupProps } from '.';
+
+const story = {
+  title: 'Example/FormGroup',
+  component: FormGroup,
+  argTypes: {
+    children: { control: { type: 'object' } },
+    className: { control: { type: 'text' } },
+    controlId: { control: { type: 'text' } },
+    validationState: { action: ['success', 'warning', 'error', null] }
```

Comment:
```suggestion
    validationState: { options: ['success', 'warning', 'error', null] }
```
This way they are a dropdown control.